optuna/artifacts/_filesystem.py

```diff
@@ -12,24 +12,7 @@
 
 
 class FileSystemArtifactStore:
-    """An artifact backend for file systems.
-
-    Example:
-       .. code-block:: python
-
-           import optuna
-           from optuna_dashboard.artifact import upload_artifact
-           from optuna_dashboard.artifact.file_system import FileSystemBackend
-
```

Comment:
Just for the record, here we removed an example from the docstring since this example uses optuna-dashboard APIs. As a follow-up, we should add an example again after merging #4823.

Reply:
Thanks for the comment. I opened https://github.com/optuna/optuna/issues/4843 to track this.